src/CMakeLists.txt

```diff
@@ -268,13 +268,8 @@ list(APPEND SM_WINDOWS_PROGRAM_DLLS
   "${SM_PROGRAM_DIR}/avcodec-53.dll"
   "${SM_PROGRAM_DIR}/avformat-53.dll"
   "${SM_PROGRAM_DIR}/avutil-51.dll"
-  "${SM_PROGRAM_DIR}/msvcp100.dll"
```

Comment:
That statement is not necessarily true. I want to say at least one of the recent releases was done on VS2010/2012. I know I did 5.0.9 for windows, but I was yelled at later for missing some things with the installer that I'm trying to look at now.

Ultimately, I agree with the idea that users should install the appropriate packages. How should they be provided if they are missing on their system?

Now, the 5_1_0 branch does require VS2013/2015, yes. Blame that on the c++1x. ;)

As an aside: this was never explained to me yet. Why does /MT work now even though windows is not building ffmpeg yet?